styles.css

```diff
@@ -8,13 +8,20 @@
 
 body {
   background-color: #fff;
-  position: absolute;
-  top: 0;
-  left: 0;
+  max-width: 375px;
+  max-height: 718px;
+}
```

Comment:
- [ ] Please give your cards an over all background color and for each card give a separate background color and also give your card `border-radius`. it is good for the visibility and better UI.
## your design 👇
![cardst](https://user-images.githubusercontent.com/66992901/193404681-f3b07aff-8645-4a20-bc5b-689aaa956005.PNG)

## Figma design👇
![cardfigmamobile](https://user-images.githubusercontent.com/66992901/193404691-6974cc94-c209-4cb9-8c19-2e14e970806f.PNG)

Reply:
Well done implementing this as requested by the previous reviewer. 👏